### DISCUSSION TITLE
v1.12.0_18-dev

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#228] [Announcements]
## Breaking

**First and foremost, no data will be lost**

You need to pull the latest server build to restore the sharing functionality.

## What's Changed
* [MOBILE] Input validation for email and server endpoint in mobile app by @matthinc in https://github.com/alextran1502/immich/pull/211
* [WEB] Added account info panel with sign-out button by @alextran1502 in https://github.com/alextran1502/immich/pull/219
* [SERVER] Fixed ENABLE_MAPBOX value is ignored by @alextran1502 in https://github.com/alextran1502/immich/pull/223
* [SERVER] Fix 500 error on login with email not in DB by @jbaez in https://github.com/alextran1502/immich/pull/212
* [MOBILE] fix: out of memory error when uploading large assets on slow internet by @zackpollard in https://github.com/alextran1502/immich/pull/224
* [MOBILE + SERVER] Refactor API for albums feature by @jbaez in https://github.com/alextran1502/immich/pull/155

## F-Droid
You can get the app on F-droid by clicking the image below.

[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
    alt="Get it on F-Droid"
    height="80">](https://f-droid.org/packages/app.alextran.immich)

#### You can also download the app from Google Play Store [here](https://play.google.com/store/apps/details?id=app.alextran.immich) 

*The App version might be lagging behind the latest release due to the review process.*

<p align="left">
  <img src="https://user-images.githubusercontent.com/27055614/164953845-4749bccf-8f9d-45e1-ac41-61fe98d6e5b4.png" width="200" title="Google Play Store">
<p/>

#### You can download the app from Apple AppStore [here](https://apps.apple.com/us/app/immich/id1613945652):

<p align="left">
  <img src="https://user-images.githubusercontent.com/27055614/164953841-de6e3f0f-4fd4-4e25-8327-6c3ccc622fc2.png" width="200" title="Apple App Store">
<p/>

*The App version might be lagging behind the latest release due to the review process.*

## Support
You can support the project using [Github Sponsor](https://github.com/sponsors/alextran1502) or [!["Buy Me A Coffee"](https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png)](https://www.buymeacoffee.com/altran1502)

**Full Changelog**: https://github.com/alextran1502/immich/compare/v1.11.0_17-dev...v1.12.0_18-dev

<hr /><em>This discussion was created from the release <a href='https://github.com/alextran1502/immich/releases/tag/v1.12.0_18-dev'>v1.12.0_18-dev</a>.</em>

**Comment by sbkg0002** [comment]
Thanks again for the great work!

Is it me, or is the f-droid repo not updated since some weeks?

**Reply by alextran1502** [reply]
Let me check, probably I have to update the metadata file after upgrade to flutter 3

**Reply by alextran1502** [reply]
F-droid was updated, but not with the latest version. It take them a while (up to 5 days) to pull the changes and update on the "store"

**Reply by sbkg0002** [reply]
Thanks, got the update this morning and works great!
